packages/commonwealth/client/scripts/views/pages/discussions/CommentCard/CommentCard.scss

```diff
@@ -60,10 +60,9 @@
       div,
       span,
       p {
-        font-size: 16px !important;
```

Comment:
16px font was a requirement for a default size of font in [comments overhaul work](https://github.com/hicommonwealth/commonwealth/pull/5129). Probably it was not ideal, because it did not respect different sizes for h1 and h2. But I think we should keep  16px for body, especially that this is a most up-to-date requirement from Jessica 
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/d6e475ca-0cf9-46ad-bafd-9bc24c7c1834)